### ISSUE TITLE
[Feature Request]: stamp  add coordinates or customize coordinates

### ISSUE THREAD

**Issue by dislazy** [Stirling-Tools/Stirling-PDF#2171]
### Feature Description

Can the stamp function add coordinates or customize coordinates?

### Why is this feature valuable?

Because I use stamp mainly to stamp PDF, but the stamp may be in different positions each time, and it needs to be determined according to the actual writing position. Fixed coordinates will make the stamp less professional. Thanks to the author

### Suggested Implementation

![image](https://github.com/user-attachments/assets/87da05c6-029b-491f-bf4b-22eb53391887)

### No Duplicate of the Feature

- [X] I have verified that there are no existing features requests similar to my request.

**Comment by thisisrenan** [comment]
Hi @dislazy, I can work on it?

**Comment by Frooodle** [comment]
Sounds more like we need to add transparency and other features of stamp to add image ?

What's the functionality of stamp which you want that is missing from the simple drag drop image?

Is it the automated x y positioning Vs UI?

**Comment by dislazy** [comment]
> Hi @dislazy, I can work on it?

yes

**Comment by dislazy** [comment]
> Sounds more like we need to add transparency and other features of stamp to add image ?
> 
> What's the functionality of stamp which you want that is missing from the simple drag drop image?
> 
> Is it the automated x y positioning Vs UI?

yes  That's what I want to do, and it's succinct to put it that way

**Comment by Frooodle** [comment]
We have the api endpoint /add-image which would do this via API for x and y co-ords
Will keep this open to add to UI side x/y placement

**Comment by jbrunton96** [comment]
This has been done for a while
